app/frontend/styles/_metadata.scss

```
@@ -0,0 +1,107 @@
+// From https://github.com/alphagov/govuk_publishing_components
```

Might be worth linking to the specific file here, to make copying across any future updates easier?

```suggestion
// From https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss
```

---

sure — or if we're making a new namespace, shall we just take that as a dependency?